impeller/aiks/canvas.cc

```diff
@@ -23,6 +23,79 @@
 
 namespace impeller {
 
+namespace {
+
+static std::shared_ptr<Contents> CreateContentsForGeometryWithFilters(
+    const Paint& paint,
+    std::shared_ptr<Geometry> geometry) {
+  auto contents = paint.color_source.GetContents(paint);
```

Comment:
https://google.github.io/styleguide/cppguide.html#Type_deduction

If we are at the level where we are trying to optimize away superfluous shared_ptr copies having the actual type makes that easier to review.  I'm not sure if a `const auto&` would be more appropriate.  Hopefully our linters find that stuff now.